inc/wp-taxonomy-landing/functions/cftl-admin.php

```diff
@@ -447,7 +447,17 @@ function cftl_tax_landing_main($post) {
 		<?php
 			//allow 'custom' if we have a single term
 			$terms = get_the_terms( $post->ID, 'series');
-			if (count($terms) == 1) $series_id = $terms[0]->term_taxonomy_id;
+			/*
+            * issue #1904
+            * when trying to check the count of false we get an error
+            * check its not false and is and array before trying to count it
+            */
+            $terms = get_the_terms( $post->ID, 'series');
```

Comment:
@rlambertsen a few things:
- can you fix the spacing here?
- `$terms` is defined twice. Once at 449 then again at 455
- add another space at after `'series'`;